package/utils/utils.go

```diff
@@ -16,6 +17,43 @@ func TransactionPanicRecover(tx *gorm.DB) {
 	}
 }
 
+func ApplyQueryParams(tx *gorm.DB, queryParams map[string][]string) *gorm.DB {
```

Comment:
As you might have understood by my previous comments, I am not the biggest fan of this function and this approach